R/utils-render.R

```diff
@@ -98,6 +98,9 @@ startTemplatedRender <- function(reportJson, author){
   #load header image
   templateData[["usgsLogoBase64"]] <- loadUsgsLogoBase64()
   
+  #load symbology processor images for derivation chain report
```

Comment:
This should not be in the shared global function. There is a place for it in derivationchain-data.R.